src/test/java/SeleniumWebSiteTest.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+
+public class SeleniumWebSiteTest {
+    @Test
+    public void testSeleniumWebDriver() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver");
+        WebDriver driver = new ChromeDriver();
+        try {
+
+            driver.manage().window().maximize();
+            driver.get("https://www.selenium.dev/");
+            driver.findElement(By.xpath("//a[contains(text(),'Downloads')]")).click();
+            WebElement latestStableVersion = driver.findElement(By.xpath("//p[contains(text(), 'Latest')]/a"));
```

Comment:
Better to have "latest stable version" text in xpath selector, so it will more obviously show what your code selects.

Reply:
I intended to, but only during class noticed I missed the rest.

Reply:
OK. Add the missed part.

Reply:
Addressed in 6cce6dc.